qiita_pet/support_files/doc/source/admin/portals.rst

```diff
@@ -0,0 +1,72 @@
+.. _portals:
+
+.. index:: portals
+
+User Portals
+============
+
+User portals allow a user to cordon off specific studies and analyses into their
+ own user interface, which we call a portal, selectively slicing the data
+ available through that portal.
+
+The main ``QIITA`` portal gives access to all studies in the database, while
+adding studies to a specific portal and changing the config file to point to the
+portal will show the same interface, but with only the selected studies. To
+change the portal being accessed, simply change the `portal` setting in the
+qiita config file. This must correspond to a portal name in the database.
+
+Creating/Deleting a portal
+--------------------------
+Portals are created using the command line interface. A portal may be added
+using the command ``qiita-env create-portal PORTAL_NAME PORTAL_DESC``.
+Conversely, a portal may be deleted by the command
+``qiita-env remove-portal PORTAL_NAME``.
+Note, however, that you can not remove a portal that has one or more
+studies/analyses attached to it. The portal must be completely connectionless
+to be deleted.
+
+Adding/Deleting studies on a portal
+-----------------------------------
+Studies will automatically be added to the main ``QIITA`` portal, as well as
+the portal set in the config file if different than ``QIITA``. To add or remove
+a study from a portal, there are two methods: Command line and web interface.
+
+Command line additions and deletions can be done using the
+``qiita db portal [add/remove]-studies PORTAL_NAME STUDY_LIST`.
```

Comment:
missing back-ticks at the end of the command. Note how `The` in the following line is also in blue.